gradle/versions.gradle

```diff
@@ -11,7 +11,7 @@ ext.libVersions = [
     jackson: '2.5.1',
     jackson_annotation: '2.5.0',
     cassandra: '2.2.1',
-    cassandra_driver_core: '2.2.0-rc3',
+    cassandra_driver_core: '2.1.5',
```

Comment:
Why this version? Latest stable 2.x is 2.1.9 released in November 2015, see https://github.com/datastax/java-driver/blob/2.1/README.md